torch/jit/frontend.py

```diff
@@ -145,7 +145,7 @@ def get_jit_class_def(cls, self_name):
     return build_class_def(ctx, py_ast.body[0], method_defs, self_name)
 
 
-def get_jit_def(fn, self_name=None):
+def get_jit_def(fn, self_name=None, def_name_override=None):
```

Comment:
def_name_override should probably not be optional or an override, but actually just the name to use everytime. It seems like otherwise, it will cause bugs when people don't realize the same thing this bug addresses: that this is the name of the field of this method. For instance frontend.py:136 is probably wrong too but wasn't caught in this change.